### PR TITLE
minor fixes

### PR DESCRIPTION
**Pull request by maboelsoudd2l**
Wanted to address small things that annoyed me. Just a mini-dragon-slayer.

Specifically:
When we press cancel in the dialog it logs errors.
I didn't like the way we reset the dialog.

### CODE REVIEW

File: components/d2l-quick-eval/d2l-quick-eval-activities.js [hunk 1]
```diff
@@ -434,6 +434,7 @@ class D2LQuickEvalActivities extends mixinBehaviors(
 		const actionDialog = this.shadowRoot.querySelector('d2l-quick-eval-action-dismiss-dialog');
 
 		return actionDialog.open().then(action => {
+			if (!action) return;
```
Comment (maboelsoudd2l) [inline components/d2l-quick-eval/d2l-quick-eval-activities.js]: if there is no action, don't try to do stuff, we are probably cancelling the dialog.

File: components/d2l-quick-eval/dismiss/d2l-quick-eval-action-dismiss-dialog.js [hunk 2]
```diff
@@ -57,12 +57,20 @@ class D2LQuickEvalActionDialog extends RtlMixin(LitQuickEvalLocalize(LitElement)
 				<p class="d2l-label-text">${this.localize('dismissUntil')}</p>
 				<div class="radio-container">
 					<label class="d2l-input-radio-label">
-						<input @change="${this._updateProp.bind(this, DISMISS_TYPES.date)}" type="radio" name="${groupID}">
+						<input
+						type="radio"
+						name="${groupID}"
+						.checked="${this.selectedRadio === DISMISS_TYPES.date}"
```
Comment (maboelsoudd2l) [inline components/d2l-quick-eval/dismiss/d2l-quick-eval-action-dismiss-dialog.js]: way better than querying the children of a radio selector

File: components/d2l-quick-eval/dismiss/d2l-quick-eval-action-dismiss-dialog.js [hunk 3]
```diff
@@ -77,14 +85,8 @@ class D2LQuickEvalActionDialog extends RtlMixin(LitQuickEvalLocalize(LitElement)
 	}
 
 	_clearSelectedOptions() {
-		this.selectedRadio = undefined;
-		this._date = undefined;
-
-		// Get the Div holding the radio buttons and clear them all
-		const elements = this.shadowRoot.querySelector('.radio-container').children;
```
Comment (maboelsoudd2l) [inline components/d2l-quick-eval/dismiss/d2l-quick-eval-action-dismiss-dialog.js]: Never been a big fan of these selectors for a couple of reasons, specially class names for a few reasons:
1. they create an implicit dependency with the stuff in the render function.
2. they create an implicit dependency for stuff in the styles.

**Comment by AlexBedley** [comment]
I like these changes but I'm hesitant to introduce more risk at this point. I think we should hold off until after the release

**Comment by maboelsoudd2l** [comment]
@AlexBedley the `if (!action) return;` is necessary in my opinion because otherwise kibana will complain about it every time someone presses cancel.

File: components/d2l-quick-eval/d2l-quick-eval-activities.js (hunk 1 above)
Comment (AlexBedley) [reply components/d2l-quick-eval/d2l-quick-eval-activities.js]: can we be certain they are cancelling the dialog?

Comment (maboelsoudd2l) [reply components/d2l-quick-eval/d2l-quick-eval-activities.js]: There are multiple buttons that close the dialog, we want any button that isn't submit, to cancel the behaviour.

**Pull request merged**